<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="echo" name="AcceptanceTests" tests="17" failures="4" timestamp="4 Jul 2014 11:06:26 GMT" time="724.481" errors="0">
  <testcase name="createOrgTest" time="54.319" classname="tests.BuildAcceptanceTests"/>
  <testcase name="loginWithFailTest" time="72.023" classname="tests.BuildAcceptanceTests"/>
  <testcase name="confirmAdminAccountTest" time="38.685" classname="tests.BuildAcceptanceTests"/>
  <testcase name="loginAsNewSuperAdminTest" time="32.47" classname="tests.BuildAcceptanceTests"/>
  <testcase name="verifyMessagesInNewsTest" time="10.708" classname="tests.BuildAcceptanceTests">
    <failure type="com.gargoylesoftware.htmlunit.ElementNotFoundException" message="elementName=[Show all] attributeName=[xpath] attributeValue=[//a[contains(text(), &amp;apos;See all&amp;apos;) and contains(text(), &amp;apos;messages ...&amp;apos;)]]">
      <![CDATA[com.gargoylesoftware.htmlunit.ElementNotFoundException: elementName=[Show all] attributeName=[xpath] attributeValue=[//a[contains(text(), 'See all') and contains(text(), 'messages ...')]]
	at objects.Element.click(Element.java:81)
	at objects.Button.click(Button.java:39)
	at pages.HQ.NewsPop.openNewsPage(NewsPop.java:18)
	at tests.BuildAcceptanceTests.verifyMessagesInNewsTest(BuildAcceptanceTests.java:171)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- verifyMessagesInNewsTest -->
  <testcase name="createSupporerManually" time="109.549" classname="tests.BuildAcceptanceTests"/>
  <testcase name="verifyLinksInSideBarTest" time="0.557" classname="tests.BuildAcceptanceTests"/>
  <testcase name="verifyLinksInToolSetTest" time="6.015" classname="tests.BuildAcceptanceTests"/>
  <testcase name="sendTestEmailsTest" time="90.99" classname="tests.BuildAcceptanceTests">
    <failure type="com.gargoylesoftware.htmlunit.ElementNotFoundException" message="elementName=[Send Now button] attributeName=[xpath] attributeValue=[//button[@id=&amp;apos;btnPublish&amp;apos;]]">
      <![CDATA[com.gargoylesoftware.htmlunit.ElementNotFoundException: elementName=[Send Now button] attributeName=[xpath] attributeValue=[//button[@id='btnPublish']]
	at objects.Element.click(Element.java:81)
	at objects.Button.click(Button.java:39)
	at pages.HQ.Emails.AddEmailsPage_PublishTab.fillAllFieldsAndPublish(AddEmailsPage_PublishTab.java:16)
	at tests.BuildAcceptanceTests.sendTestEmailsTest(BuildAcceptanceTests.java:190)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- sendTestEmailsTest -->
  <testcase name="sendEmailsTest" time="52.359" classname="tests.BuildAcceptanceTests"/>
  <testcase name="createCMManually" time="5.636" classname="tests.BuildAcceptanceTests">
    <failure type="com.gargoylesoftware.htmlunit.ElementNotFoundException" message="elementName=[Invite new user] attributeName=[xpath] attributeValue=[//button[text()=&amp;apos;+ invite&amp;apos;]]">
      <![CDATA[com.gargoylesoftware.htmlunit.ElementNotFoundException: elementName=[Invite new user] attributeName=[xpath] attributeValue=[//button[text()='+ invite']]
	at objects.Element.click(Element.java:81)
	at objects.Button.click(Button.java:39)
	at pages.HQ.Manage.AccountsPage.openInviteNewUserPage(AccountsPage.java:14)
	at tests.BuildAcceptanceTests.createCMManually(BuildAcceptanceTests.java:258)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- createCMManually -->
  <testcase name="creteSegmentTest" time="14.286" classname="tests.BuildAcceptanceTests">
    <failure type="com.gargoylesoftware.htmlunit.ElementNotFoundException" message="elementName=[Element was not found] attributeName=[xpath] attributeValue=[//div[@class=&amp;apos;ruleSet&amp;apos;]/descendant::div[@class=&amp;apos;row rule ng-scope&amp;apos;][1]/descendant::span[text()=&amp;apos;Choose metric&amp;apos;]]">
      <![CDATA[com.gargoylesoftware.htmlunit.ElementNotFoundException: elementName=[Element was not found] attributeName=[xpath] attributeValue=[//div[@class='ruleSet']/descendant::div[@class='row rule ng-scope'][1]/descendant::span[text()='Choose metric']]
	at objects.Element.findElementByXpath(Element.java:395)
	at objects.DropDown.selectByLabel(DropDown.java:31)
	at pages.HQ.Segments.SegmentsAddPage.selectChooseMetric(SegmentsAddPage.java:87)
	at pages.HQ.Segments.SegmentsAddPage.addRule(SegmentsAddPage.java:48)
	at pages.HQ.Segments.SegmentsAddPage.createNewSegment(SegmentsAddPage.java:31)
	at tests.BuildAcceptanceTests.creteSegmentTest(BuildAcceptanceTests.java:143)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- creteSegmentTest -->
  <testcase name="confirmTestEmailsWereSentTest" time="0.0" classname="tests.BuildAcceptanceTests">
    <skipped/>
  </testcase> <!-- confirmTestEmailsWereSentTest -->
  <testcase name="confirmEmailsWereSentTest" time="23.644" classname="tests.BuildAcceptanceTests"/>
  <testcase name="confirmCMAccountTest" time="0.0" classname="tests.BuildAcceptanceTests">
    <skipped/>
  </testcase> <!-- confirmCMAccountTest -->
  <testcase name="verifyMessagesInAlertsTest" time="0.0" classname="tests.BuildAcceptanceTests">
    <skipped/>
  </testcase> <!-- verifyMessagesInAlertsTest -->
  <testcase name="loginAsNewCMTest" time="0.0" classname="tests.BuildAcceptanceTests">
    <skipped/>
  </testcase> <!-- loginAsNewCMTest -->
</testsuite> <!-- AcceptanceTests -->
